src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -81,12 +81,16 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
             var buildOpId = logger.StartOperation(appName);
 
             var buildInfo = new DefinitionListFormatter();
-            buildInfo.AddDefinition("Build Operation ID", buildOpId);
             buildInfo.AddDefinition("Oryx Version", $"{Program.GetVersion()}, Commit: {Program.GetCommit()}");
+            buildInfo.AddDefinition("Oryx Repository", "https://github.com/microsoft/oryx");
```

Comment:
This is how looks like, FYI:
![image](https://user-images.githubusercontent.com/6559784/54715715-5723c500-4b11-11e9-8563-b58431d9d9cc.png)

Reply:
I'd use a more free form text, which also explains what Oryx is, e.g. `Build orchestrated by Microsoft Oryx, https://github.com/Microsoft/Oryx/. You can report issues at 'https://github.com/Microsoft/Oryx/Issues'`.
What do you think?

Reply:
cc @joshgav

Reply:
Updated. Latest looks like below:

![image](https://user-images.githubusercontent.com/6559784/54726499-a62d2280-4b30-11e9-84a2-6e379d2904d5.png)